apps/heft/src/cli/actions/CleanAction.ts

```diff
@@ -116,5 +117,7 @@ export class CleanAction extends CommandLineAction implements IHeftAction {
 
     // Delete the files
     await deleteFilesAsync(deleteOperations, this._terminal);
+
+    return deleteOperations.length === 0 ? OperationStatus.NoOp : OperationStatus.Success;
```

Comment:
Doesn't look like this is being used?

Reply:
`runWithLoggingAsync` expects the return value to be of type `OperationStatus` now.